src/core/bpf-lsm.c

```diff
@@ -125,13 +125,15 @@ static int mac_bpf_use(void) {
         }
 }
 
-bool lsm_bpf_supported(void) {
+bool lsm_bpf_supported(bool initialize) {
```

Comment:
Maybe `cache_only` or something like that instead, to reflect what it does rather than what calls it? Just a nit, not too important

Reply:
So for me, "initialize" **is** about what it does: it answers the question "should the function do initialization?", and the answer to that is provided by the caller. The fact that there's some caching going on underneath is an implementation detail of the function.